src/CppUTest/Utest.cpp

```diff
@@ -486,9 +488,13 @@ void UtestShell::print(const SimpleString& text, const char* fileName, int lineN
     print(text.asCharString(), fileName, lineNumber);
 }
 
+extern "C++" {
+
```

Comment:
Gcc get's a little confused babout the linkage of static members, but hey, we iron it out :P